comptests/prov/test_prov_k8s_cortx_upgrade.py

```diff
@@ -269,14 +268,84 @@ def test_32605(self):
         LOGGER.info("Step 5: Done.")
 
         LOGGER.info("Step 6: Test parallel upgrades.")
+        que = queue.Queue()
         parallel_upgrade_message = "An upgrade is already in progress"
-        upgrade_thread = Thread(target=self.perform_upgrade)
-        parallel_upgrade_thread = Thread(target=self.perform_upgrade, args=(False,))
+        upgrade_thread = Thread(target=self.perform_upgrade, args=(True, que,))
+        parallel_upgrade_thread = Thread(target=self.perform_upgrade, args=(False, que,))
         upgrade_thread.start()
         time.sleep(10)  # Wait to start upgrade_thread thread
         parallel_upgrade_thread.start()
-        resp = self.que.get(1)
+        resp = que.get(1)
         assert_utils.assert_in(parallel_upgrade_message, resp[1])
         upgrade_thread.join()
         LOGGER.info("Step 6: Done.")
         LOGGER.info("Test Completed.")
+
+    @pytest.mark.lc
+    @pytest.mark.comp_prov
+    @pytest.mark.tags("TEST-37514")
+    def test_37514(self):
+        """
+        Verify Upgrade script throws an error message when invalid argument is passed.
+        """
+        LOGGER.info("Test Started.")
+        LOGGER.info("Check proper error message when invalid argument passed to upgrade script.")
+        upgrade_cmd = "sh upgrade-cortx-cloud.sh -abc"
+        cmd = "cd {}; {}".format(self.prov_deploy_cfg["git_remote_path"], upgrade_cmd)
```

Comment:
can we add this in command.py file

Reply:
Hi @pragamj ,

This is specific to this TC only. Do we want to add in commands.py?

Reply:
Ok. I'll add these to commands.py file.

Reply:
Or else we can add these to prov specific yaml file, i.e., test_prov_config.yaml? Either yaml file or commands.py please suggest.

Reply:
for now I have kept them in test_prov_config.yaml file as these are particular to these TCs only.

Reply:
better to use commands.py as we have all deployment and upgrade commands in their.

Reply:
Done. Added in commands.py file.